### DISCUSSION TITLE
How to deal with infrastructures claiming to be carbon neutral ?

### DISCUSSION THREAD

**Discussion by gmaze** [green-coding-solutions/green-metrics-tool#796] [General]
Hi there,

I wonder also how could you deal with infrastructures claiming to be carbon neutral ?

This is what github is claiming: https://github.blog/2021-04-22-environmental-sustainability-github/

It would be nice to consider being able to label, and account differently, in the CarbonDB dashboard CO2 emissions emitted from carbon neutral infra, vs those that are not.

Honestly I don't buy the github claim, but some CarbonDB reporting could indeed really be compensated

**Comment by ribalba** [comment]
Hey,

thanks for reaching out. Yes this is something we have and are discussing constantly. The main problem is that pretty much everyone claims to be carbon neutral by buying CO2 certificates somewhere or "green" power agreements. No computation is carbon neutral! You need to produce the hardware, you need to build the infrastructure and even if you use 100% solar you still need to build the panels. So if you use a service you create carbon. Pretty much everything we do does, even breathing, so obviously it is a "how much" thing. The problem is, that we from the outside don't know what GitHub is really doing, they don't publish numbers and don't give us detailed metrics. Something we (and many more) have been asking to get for ages. So we decided to take the only value we can sort of get reliable and that is the grid intensity of the location we think the datacenter is located in. Even getting the exact location is quite hard. The best thing IMHO is if GitHub would give us these values but like most big companies they just green wash their numbers :( I hope this sort of helps. 

More context here: https://github.com/green-coding-solutions/eco-ci-energy-estimation/issues/54

**Reply by gmaze** [reply]
Fully agree !
Having the "how much" right is already a huge task, as the Green Coding work/tools show.
It's surely a good think that the scope and objective are precisely defined (focus on "how much"), as I understand that the CarbonDB is not intended to provide a formal carbon budget.

**Comment by gmaze** [comment]
So, basically the answer to the question of this thread is:

"Beyond the scope of Green Coding overall objectives"
